### ISSUE TITLE
Workbench load up error on a MacOS with ZSH - Error Code 137

### ISSUE THREAD

**Issue by nlauchande** [PacktPublishing/Machine-Learning-Engineering-with-MLflow#1]
=> ERROR [gradflow/workbench/jupyter:0.1.0 jupyter 2/10] RUN conda install --quiet --yes 'mlflow='1.13.* 'psycopg2=2 261.0s
------
> [gradflow/workbench/jupyter:0.1.0 jupyter 2/10] RUN conda install --quiet --yes 'mlflow='1.13.* 'psycopg2=2.8.*' 'conda-build' 'tqdm' 'jupyter_contrib_nbextensions' 'jupyter_nbextensions_configurator' && conda build purge-all && fix-permissions /opt/conda:
#11 6.869 Collecting package metadata (current_repodata.json): ...working... done
#11 50.57 Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
#11 50.57 Collecting package metadata (repodata.json): ...working... done
#11 212.9 Solving environment: ...working... Killed
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c conda install --quiet --yes 'mlflow='$MLFLOW_VERSION 'psycopg2=2.8.*' 'conda-build' 'tqdm' 'jupyter_contrib_nbextensions' 'jupyter_nbextensions_configurator' && conda build purge-all && fix-permissions $CONDA_DIR]: exit code: 137
make: *** [build] Error 17

**Comment by nlauchande** [comment]
After many tries, I was able to replicate it when I reduced the memory of my docker image in docker to 2GB.

When I upped it to 4GB it worked.

Found more insights on this article : https://betterprogramming.pub/understanding-docker-container-exit-codes-5ee79a1d58f6

"
E"xit Code 137
This indicates that container received SIGKILL
A common event that initiates a SIGKILL is a docker kill. This can be initiated either manually by user or by the docker daemon:
docker kill <container-id>
docker kill can be initiated manually by the user or by the host machine. If initiated by host machine, then it is generally due to being out of memory. To confirm if the container exited due to being out of memory, verify docker inspect against the container id for the section below and check if OOMKilled is true (which would indicate it is out of memory):
"State": {
 "Status": "exited",
 "Running": false,
 "Paused": false,
 "Restarting": false,
 "OOMKilled": true,
 "Dead": false,
 "Pid": 0,
 "ExitCode": 137,
 "Error": "",
 "StartedAt": "2019-10-21T01:13:51.7340288Z",
 "FinishedAt": "2019-10-21T01:13:51.7961614Z"
}
